sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1.scala

```diff
@@ -57,7 +57,7 @@ case class DisplayWorkV1(
       "Relates a work to the general thesaurus-based concept that describes the work's content."
   ) subjects: List[DisplayConcept] = List(),
   @ApiModelProperty(
-    dataType = "uk.ac.wellcome.display.models.DisplayConcept",
+    dataType = "List[uk.ac.wellcome.display.models.DisplayConcept]",
```

Comment:
I’d double check, but we may be able to get rid of annotations where it’s a List[…] type – we override them in certain places if it’s an Option[…] or Option[List[…]] so the optional-ness doesn’t appear in the API, but the auto-generated values might be okay here.

Reply:
Uhm ok, I did have a quick look and we seem to override for lists everywhere, but it migh not be needed